### ISSUE TITLE
CartoDB not working

### ISSUE THREAD

**Issue by masterflorin** [tillnagel/unfolding#130]
I tried to do an example with CartoDB but seeing that it's labeled under not working, I tried to try another way but then I got that the API is deprecated.

Is it still not usable? Or I didn't use it properly?

**Comment by tillnagel** [comment]
Yes, still not working. And alas, no plans to make it so in the near future.

Would be happy if you manage to create a working version, and share it with us.

**Comment by masterflorin** [comment]
OK, good to know.
Well, I'll look into it as it would be an interesting challenge. :)
